javascript/react/fundamentals/the-react-component.md

```diff
@@ -27,58 +27,77 @@ aspects:
 
 ---
 
-# The **React** component
+# The React component
 
 ---
 ## Content
 
-In **React** a `component` represents a fundamental *class* you can work with to build re-usable and independent `UI` blocks.
+In **React**, a component represents a fundamental *class* you can work with to build re-usable and independent `UI` blocks.
 
-These `component`s behave like simple functions, they can take input arguments called `props` and will return what should be *rendered*.
+These components behave like simple functions, they can take input arguments called `props` and will return what should be *rendered*.
 
-The *rendered* output needs to be specified inside a function called `render()`.
+If you want to create a simple component, you can use a JavaScript function like this:
 
-To create a component you can `extend` the `ES6`[1] class `React.Component`:
+```jsx
+// functional component
+function Enki(props) {
+  return <p>Enki aka Ea</p>;
+}
 ```
+
+At the same time, you can also `extend` the `ES6`[1] class `React.Component` to create a component. In this case, the *rendered* output needs to be returned from a method called `render()`:
+
+```jsx
+// class component
 class Enki extends React.Component {
   render() {
     return <p>Enki aka Ea</p>;
   }
 }
 ```
 
-You can pass dynamic values to a component via `props` which will be discussed next. In addition, you can use other components when `render`ing.
+The differences between functional and class components will be discussed at the end of this workout, but for now know that, when viewed externally, each way of writing a component can render the same result.
 
-`Component` names should start with a capital letter such that you can differentiate them from **DOM tags**.
+You can pass dynamic values to a component via `props` which will be discussed next. 
+
+Component names should start with a capital letter such that you can differentiate them from **DOM tags**.
```

Comment:
Maybe explain a but more that this is a suggested convention but not a strict rule?

Reply:
oh, I always thought this was a strict rule. I'll rectify this